modal/cli/secret.py

```diff
@@ -43,12 +43,13 @@ async def list(env: Optional[str] = ENV_OPTION, json: Optional[bool] = False):
     display_table(column_names, rows, json, title=f"Secrets{env_part}")
 
 
-@secret_cli.command("create", help="Create a new secret")
+@secret_cli.command("create", help="Create a new secret. Use `--force` to overwrite any existing one.")
```

Comment:
We can add this to the option doc

Reply:
I don't think boolean options support documentation – I wasn't able to get it working and at least [one Github Isssue](https://github.com/tiangolo/typer/issues/493) suggests this is a missing feature in typer. Could be wrong though.

Reply:
Hm this worked for me here: https://github.com/modal-labs/modal-client/blob/main/modal/cli/token.py#L18-L21

Reply:
I updated it! https://github.com/modal-labs/modal-client/pull/1344